### ISSUE TITLE
White space beneath parallax elements.

### ISSUE THREAD

**Issue by Bccorb** [pixelcog/parallax.js#96]
I have implemented a simple parallax effect using this framework. 

<div class="container-fluid">
    	`<div class="row imagebg" data-parallax="scroll" data-image src="img/ServicesPageBackground.jpg">`
` ** code here **`
`</div>`
`</div>`

The problem is for some reason there is a little bit of white space that is created at the bottom of the div. 

Don't know if my scroll spy is doing it or what is there a way to fix this? 

Take a look, here is the site I am building. http://EvnaDesign.com

**Comment by atavernor** [comment]
It is just the image scaling. If you can live with cover scaling this will ensure that the available space is always filled without distortion of the aspect ratio.
``.about .imagebg{
background-size:cover
}``

**Comment by Bccorb** [comment]
@atavernor Even with that properties the small bit of white space persists! :(

**Comment by atavernor** [comment]
It works when I change it live on your page. It would also be worth adding some bleed though to prevent the white flickering as you scroll.

<img width="780" alt="screen shot 2015-11-20 at 01 11 34" src="https://cloud.githubusercontent.com/assets/8500380/11289338/b63b04f2-8f23-11e5-8a9c-eb1c1886fe3d.png">

**Comment by Bccorb** [comment]
Okay, sorry I was so slow to respond let me try again and add in a little bleed! I will let you know shortly!

**Comment by Bccorb** [comment]
So the the bleed has corrected the scroll jumping, but nothing for the white space. If you re-size or refresh the screen the second time around it works, but if you go to another page and come back it does not work. Still have a few pixels of white space. 

I changed all the styles to `background-size: cover;`

**Comment by mikegreiling** [comment]
If you're seeing any issue in which the parallax background appears to be off by a bit and then snaps into place once you resize the window, that is 99% of the time caused when your layout is changed after the parallax calculation has been done.  For instance, if your page has a javascript onload method which adjusts the layout or position of the parallax window in any way after the first rendering, the parallax effect will still think it is at the old location until it is told to recalculate (this happens automatically when the window is resized).

You can trigger this manually with a call to `jQuery(window).trigger('resize').trigger('scroll');` after your layout-altering event is complete.

**Comment by mikegreiling** [comment]
@Bccorb I believe if you add that line below your `$("#navbar").affix(....` line, it'll fix the issue

**Comment by Bccorb** [comment]
Hello! Sorry for not coming back to close this. @mikegreiling your solution worked. I just had to modify that trigger into a more proper function to get the resize to fire on scroll. Thank you very much!
